modules/navier_stokes/test/tests/finite_volume/wcns/enthalpy_equation/saline_custom.prp

```diff
@@ -0,0 +1,4 @@
+// System      , id , Mol Mass , Mol Frac             , Melt(K) , Var(%) , Reference   , Boil(K) , Var(%) , Reference      , rho_a     , rho_b     , range(K)      , Var(%) , Reference     , mu1_a     , mu2_b     , mu2_a      , mu2_b      , mu2_c     , range(K)      , Var(%) , Reference    , k_a        , k_b        , range(K)     , Var(%) , Reference   , cp_a      , cp_b       , cp_c       , cp_d      , Var(%) , Reference
```

Comment:
I think you can remove this, or add a test that uses saline.

Reply:
removed thanks